perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -240,6 +240,27 @@ sub split_name {
     return { first => $first || '', last => $last || '' };
 }
 
+sub permissions {
+    my ($self, $body_id) = @_;
+
+    if ($self->is_superuser) {
+        # XXX Not great having this hardcoded here.
```

Comment:
Not a big fan of this hardcoding either. Could we pass in `$c` and build the hash from `$c->cobrand->available_permissions`?